v0/destinations/af/transform.js

```diff
@@ -56,7 +56,7 @@ function responseBuilderSimple(payload, message, destination) {
   const updatedPayload = {
     ...payload,
     eventTime: message.timestamp,
-    customer_user_id: message.user_id,
+    customer_user_id: message.userId,
```

Comment:
Should support both for backward compatibility?

Reply:
Actually the key `user_id` is obsolete now. So no events will come with that key. So ignoring this as we discussed